sql-functions/src/test/java/com/ververica/platform/sql/functions/ListAggITCase112.java

```diff
@@ -0,0 +1,129 @@
+package com.ververica.platform.sql.functions;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import org.apache.flink.api.common.restartstrategy.RestartStrategies;
+import org.apache.flink.contrib.streaming.state.RocksDBStateBackend;
+import org.apache.flink.runtime.state.StateBackend;
+import org.apache.flink.runtime.state.memory.MemoryStateBackend;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+import org.apache.flink.table.planner.factories.TestValuesTableFactory;
+import org.apache.flink.types.Row;
+import org.apache.flink.types.RowKind;
+import org.junit.Before;
+import org.junit.Test;
+
+/** Integration test for the built-in <tt>LISTAGG</tt> function. */
+public class ListAggITCase112 {
+
+  protected StreamExecutionEnvironment env;
+  protected StreamTableEnvironment tEnv;
+
+  @Before
+  public void setUp() {
```

Comment:
If you use the MiniClusterResource these tests will run much more quickly.

Reply:
Thanks for this tip. I also took the opportunity to reduce code duplication in getting the result list.